### PR TITLE
Updated Hero Section

### PR DESCRIPTION
**Pull request by aryab2003**
I have updated the  Hero section. Please review @Shurtu-gal

### CODE REVIEW

**Comment by Shurtu-gal** [comment]
Your prettier check is failing @aryab2003.

**Comment by ashutosh-rath02** [comment]
- [ ] Place the component under playground for review.

- [ ] Remove unwanted declarations as they are failing Lint checks.

**Comment by aryab2003** [comment]
> * [ done]  Place the component under playground for review.
> * [ ]  Remove unwanted declarations as they are failing Lint checks.

**Comment by aryab2003** [comment]
> Your prettier check is failing @aryab2003.

Done

**Comment by aryab2003** [comment]
@Lucif3r-in  after importing in playground, this issue is showing..help me resolve this..

**Comment by ashutosh-rath02** [comment]
image
There is some issue in your imports. Please check that. @aryab2003

File: src/components/molecules/Hero/Hero.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                    alt="Small Image"/>
+            </HeroContent>
+        </HeroWrapper>
+    );
+};
+
+export default Hero;
+
+const HeroWrapper = styled.div `
+  background-image: url(${wallpaper});
```
Comment (Shurtu-gal) [inline src/components/molecules/Hero/Hero.jsx]: Why are you putting the same styles in two places?

File: src/components/molecules/Hero/Hero.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import img from './img.png';
```
Comment (Shurtu-gal) [inline src/components/molecules/Hero/Hero.jsx]: change the name to heroBackgroundImage or sth

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

File: src/components/molecules/Hero/Hero.jsx (hunk 2 above)
Comment (aryab2003) [reply src/components/molecules/Hero/Hero.jsx]: ok

File: src/components/molecules/Hero/Hero.jsx (hunk 1 above)
Comment (aryab2003) [reply src/components/molecules/Hero/Hero.jsx]: Changed

File: src/components/molecules/Hero/Hero.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import sth from './img.png';
+import {HeroWrapper, HeroContent} from './styles';
+
+const Hero = () => {
+    return (<HeroWrapper>
+        <HeroContent>
+            <img src={sth}
```
Comment (Shurtu-gal) [inline src/components/molecules/Hero/Hero.jsx:8]: @aryab2003 Code it don't just put the image.

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

File: src/components/molecules/Hero/Hero.jsx (hunk 3 above)
Comment (aryab2003) [reply src/components/molecules/Hero/Hero.jsx:8]: ok..can you help  as to how would i go about doing that?

Comment (Shurtu-gal) [reply src/components/molecules/Hero/Hero.jsx:8]: Write the text as headings and use button components.

Comment (aryab2003) [reply src/components/molecules/Hero/Hero.jsx:8]: ok

**Comment by Shurtu-gal** [comment]
Closing this.

**Pull request closed without merging**